massa-ledger-exports/src/ledger_changes.rs

```diff
@@ -248,9 +249,8 @@ impl LedgerEntryUpdateDeserializer {
                 Included(Amount::MIN),
                 Included(Amount::MAX),
             )),
-            bytecode_deserializer: SetOrKeepDeserializer::new(VecU8Deserializer::new(
-                Included(u64::MIN),
-                Included(max_datastore_value_length),
+            bytecode_deserializer: SetOrKeepDeserializer::new(BytecodeDeserializer::new(
+                max_datastore_value_length + 4,
```

Comment:
No hardcoding...

Reply:
Ok, should I add `const MAX_VERSION_BYTE_SIZE: u64 = 4` in Bytecode?
Or should I just take the max_datastore_value_length as arg and hide the magic value in the BytecodeDeserializer code?

Reply:
If I'm not mistaken you use a u64 for version and the size of it is 8 no?

Reply:
Ah indeed :)

Reply:
As the max_datastore_value_length only covers the inner `vec<u8>` length, the magic value is not needed, we deserialize the version bytes beforehand. Fixed in https://github.com/massalabs/massa/pull/3603/commits/c5dbdd89580157213f6ff4d7ef0e53311d8e88e0